zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSettingManager.java

```diff
@@ -888,18 +886,16 @@ public void removeRepository(String id) throws IOException {
   }
 
   public void removeNoteInterpreterSettingBinding(String user, String noteId) throws IOException {
-    synchronized (interpreterSettings) {
```

Comment:
@Leemoonsoo , Thanks for the review. Is there anything that I need to do  to resolve this ?